Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Make sure gcc et al are present for native linking
+FROM oracle/graalvm-ce:20.2.0-java11
+
+# Intended for: zip -r -j target/scala-2.13/invoicing-api.zip target/scala-2.13/bootstrap
+RUN yum -y install zip
+
+# Install sbt and coursier via www.scala-lang.org/2020/06/29/one-click-install.html
+RUN curl -Lo /opt/cs https://git.io/coursier-cli-linux && chmod +x opt/cs
+RUN echo 'Y' | ./opt/cs setup
+ENV PATH="/root/.local/share/coursier/bin:$PATH"
+
+# Manage JVM with coursier
+RUN cs java-home --jvm graalvm-java11:20.2.0
+
+# Populate sbt cache to speed up build
```

Comment:
Does this have a big impact on build time compared to letting sbt just build the project normally? If we were to revisit this in six months would having to understand this still be justified?

Reply:
giter8 param can be provided like so

```
sbt new scala/scala-seed.g8 --name=sbt-cache
```

I will remove this as it is just an optimisation which has an effect only first time container is launched. Most of the time developer will just keep the container running during development so shaving off less than a minute once does not seem to warrant the confusion.